web/src/condense.ts

```diff
@@ -290,10 +317,21 @@ export function initialize() {
     $("#message_feed_container").on("click", ".message_condenser", function (e) {
         const $row = $(this).closest(".message_row");
         const id = rows.id($row);
+        if (typeof id !== "number") {
+            return;
+        }
+
         // Focus on the condensed message.
-        assert(message_lists.current !== undefined);
-        message_lists.current.select_id(id);
-        message_lists.current.get(id).condensed = true;
+        if (message_lists.current !== undefined) {
```

Comment:
although, I am not sure why you removed the old assert syntax and implemented nested `if-else` statements. let's wait for feedback from core team.

Reply:
@Mahhheshh Maybe I shouldn't have to use an if-else. But there is a type error despite the assertation.

Reply:
assert is more correct for cases we consider impossible.

Reply:
@timabbott  Yes, the problem is with `.get()`. 

Okay, noted.